lib/pages/home/wallet/wallet.dart

```diff
@@ -57,12 +62,19 @@ class WalletPage extends StatelessWidget {
     );
   }
 
-  Widget _buildCurrencyRow(BuildContext context, Currency currency) {
+  Widget _buildCurrencyRow(BuildContext context, int index, Currency currency) {
+    Color currencyRowColour = Colors.white12;
+    Color highlightColor = Theme.of(context).brightness ==  Brightness.light ? Colors.black12 : Colors.white12;
+    currencyRowColour = (index % 2 != 0 ? currencyRowColour = highlightColor : currencyRowColour = Colors.transparent);
     return Container(
-      margin: EdgeInsets.all(8.0),
+      margin: EdgeInsets.all(0.0),
+      padding: EdgeInsets.only(left: 0.0, right: 8.0, top: 8.0, bottom: 8.0),
+      color: currencyRowColour,
```

Comment:
Code can be a bit neater. Change the code at the top to just this:
```
Color highlightColor = Theme.of(context).brightness ==  Brightness.light ? Colors.black12 : Colors.white12;
```
Then when deciding the Container color, do this:
```
color: index % 2 != 0 ? highlightColor : null,
```

Reply:
Updated